scripts/install

```diff
@@ -57,11 +52,13 @@ function hindsight {
         local -r set_values="--set $vals"
     fi
 
-    deps
-
     info "Installing Hindsight"
-    helm upgrade ${release} ./helm \
+    helm repo add inhindsight https://inhindsight.github.io/charts
```

Comment:
Is this the correct URL? It's 404 for me although I can reach https://github.com/inhindsight/charts

Reply:
it is, theres. is. an implicit index.yaml for chart repos so the url is https://inhindsight.github.io/charts/index.yaml

Reply:
Lol. I asked the exact same question on the last PR